packages/signia/src/index.ts

```diff
@@ -6,15 +6,72 @@
  * @packageDocumentation
  */
 
-export { atom, isAtom } from './Atom'
-export type { Atom, AtomOptions } from './Atom'
-export { unsafe__withoutCapture, whyAmIRunning } from './capture'
-export { computed, getComputedInstance, isUninitialized, withDiff } from './Computed'
-export type { Computed, ComputedOptions } from './Computed'
-export { EffectScheduler, react, reactor } from './EffectScheduler'
-export type { Reactor } from './EffectScheduler'
-export { EMPTY_ARRAY } from './helpers'
-export { isSignal } from './isSignal'
-export { transact, transaction } from './transactions'
-export { RESET_VALUE } from './types'
-export type { Signal } from './types'
+export type { Atom, AtomOptions } from './Atom.js'
+export type { Computed, ComputedOptions } from './Computed.js'
+export type { Reactor } from './EffectScheduler.js'
+export type { Signal } from './types.js'
+
+import { atom, isAtom } from './Atom.js'
+import { unsafe__withoutCapture, whyAmIRunning } from './capture.js'
+import { computed, getComputedInstance, isUninitialized, withDiff } from './Computed.js'
+import { EffectScheduler, react, reactor } from './EffectScheduler.js'
+import { EMPTY_ARRAY } from './helpers.js'
+import { isSignal } from './isSignal.js'
+import { transact, transaction } from './transactions.js'
+import { RESET_VALUE } from './types.js'
+
+/**
+ * @public
+ */
+const signia: any = {
```

Comment:
this probably shouldn't be `any`
```suggestion
const signia = {
```

Reply:
My VS Code was complaining without it - might have been my defaults, but you may want to verify.

(I was trying to avoid having to have both individual exports and the default export there, but since there was a test case already in case testing for the default export, I ended up adding that solution.)